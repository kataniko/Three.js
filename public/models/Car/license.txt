Model Information:
* title:	Huracan EAGLE™️
* source:	https://sketchfab.com/3d-models/huracan-eagletm-ea7b91e22b8848ac8d02c79b1c574fc1
* author:	SDC PERFORMANCE™️ (https://sketchfab.com/3Duae)

Model License:
* license type:	CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)
* requirements:	Author must be credited. Commercial use is allowed.

If you use this 3D model in your project be sure to copy paste this credit wherever you share it:
This work is based on "Huracan EAGLE™️" (https://sketchfab.com/3d-models/huracan-eagletm-ea7b91e22b8848ac8d02c79b1c574fc1) by SDC PERFORMANCE™️ (https://sketchfab.com/3Duae) licensed under CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)