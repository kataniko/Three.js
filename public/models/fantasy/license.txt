Model Information:
* title:	fantasy green garden space baked
* source:	https://sketchfab.com/3d-models/fantasy-green-garden-space-baked-56ad966c265243ad92716195a29164a6
* author:	McWinterL (https://sketchfab.com/winterlea)

Model License:
* license type:	CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)
* requirements:	Author must be credited. Commercial use is allowed.

If you use this 3D model in your project be sure to copy paste this credit wherever you share it:
This work is based on "fantasy green garden space baked" (https://sketchfab.com/3d-models/fantasy-green-garden-space-baked-56ad966c265243ad92716195a29164a6) by McWinterL (https://sketchfab.com/winterlea) licensed under CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)