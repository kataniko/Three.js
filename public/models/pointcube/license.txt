Model Information:
* title:	Point Cube
* source:	https://sketchfab.com/3d-models/point-cube-0da83e60b69f458dba69fb3013ef3f38
* author:	V크 V천 (https://sketchfab.com/vuvo)

Model License:
* license type:	CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)
* requirements:	Author must be credited. Commercial use is allowed.

If you use this 3D model in your project be sure to copy paste this credit wherever you share it:
This work is based on "Point Cube" (https://sketchfab.com/3d-models/point-cube-0da83e60b69f458dba69fb3013ef3f38) by V크 V천 (https://sketchfab.com/vuvo) licensed under CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)