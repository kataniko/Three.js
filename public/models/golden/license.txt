Model Information:
* title:	Golden NFT Card in a flashy display scene
* source:	https://sketchfab.com/3d-models/golden-nft-card-in-a-flashy-display-scene-581c053865e743ccb49f3f1883d5f66c
* author:	Jesse21 (https://sketchfab.com/JesseTwentyOne)

Model License:
* license type:	CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)
* requirements:	Author must be credited. Commercial use is allowed.

If you use this 3D model in your project be sure to copy paste this credit wherever you share it:
This work is based on "Golden NFT Card in a flashy display scene" (https://sketchfab.com/3d-models/golden-nft-card-in-a-flashy-display-scene-581c053865e743ccb49f3f1883d5f66c) by Jesse21 (https://sketchfab.com/JesseTwentyOne) licensed under CC-BY-4.0 (http://creativecommons.org/licenses/by/4.0/)